app/helpers/host_description_helper.rb

```diff
@@ -136,6 +136,7 @@ def base_host_overview_buttons(host)
       { :button => link_to_if_authorized(_("Audits"), { :controller => 'audits', :action => 'index', :search => "host = #{host.name}" }, :title => _("Host audit entries"), :class => 'btn btn-default'), :priority => 100 },
       ({ :button => link_to_if_authorized(_("Facts"), hash_for_host_facts_path(:host_id => host), :title => _("Browse host facts"), :class => 'btn btn-default'), :priority => 200 } if host.fact_values.any?),
       ({ :button => link_to_if_authorized(_("Reports"), hash_for_host_config_reports_path(:host_id => host), :title => _("Browse host config management reports"), :class => 'btn btn-default'), :priority => 300 } if host.reports.any?),
+      { :button => link_to(_("New page"), '#', :title => _("Switching to the new experimental host details page"), :onclick => "tfm.nav.pushUrl('#{host_details_page_path(host.name)}')", :class => 'btn btn-default'), :priority => 50 },
```

Comment:
Just curious: Why do we have the button here and not on the toolbar?

Reply:
The toolbar has been considered as an option, but this place is chosen, I don't have a strong opinion about this.